__test__/components/pages/home/Navbar.test.jsx

```diff
@@ -0,0 +1,16 @@
+import { render, screen, cleanup } from "@testing-library/react";
+import Navbar from "@/components/pages/home/Navbar";
+
+describe("Navigation Bar (Navbar) unit test", () => {
+  afterEach(cleanup);
+
+  it("valid navbar", () => {
```

Comment:
Untuk penamaan test di anak `it` harus lebih lengkap dan mewakili test didalamnya.
contohnya `Navbar should be showing valid items`.